### ISSUE TITLE
Discord isnt recognizing it

### ISSUE THREAD

**Issue by Stebulous** [Pizzabelly/EasyRP#1]
I add it to my games thing and the rich presence isnt working.

**Comment by GeorgeCY2** [comment]
is the client id right as it looks?

**Comment by GlitchMasta47** [comment]
Check your Client ID, and make sure you have correctly set up your config and your Rich Presence.

**Comment by HunterOfBugs** [comment]
Yeah, the same thing is happening to me. But It doesn't quite work when I add the game. I made sure everything was set up right in the config thing, but it's not working! Please help! REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE!!!!!!!!!!!!!!!!

**Comment by ryukora** [comment]
> Yeah, the same thing is happening to me. But It doesn't quite work when I add the game. I made sure everything was set up right in the config thing, but it's not working! Please help! REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE!!!!!!!!!!!!!!!!

check your config or application if it doesn't work tell me
